src/Themes.ts

```diff
@@ -1,226 +1,319 @@
-import { player } from './Synergism';
 import { DOMCacheGetOrSet } from './Cache/DOM';
 
-export const toggleTheme = (setting = true) => {
-    const themeButton = DOMCacheGetOrSet('theme');
-    const current = themeButton.textContent;
-
-    if (current === 'Dark Mode') { //Switches to 'Darker Mode'
-        DOMCacheGetOrSet('singularitybtn').style.backgroundColor = '#171717'; //One time, until auto will be added
-        document.body.style.setProperty('--bg-color', '#0c0c0f');
-        document.body.style.setProperty('--alert-color', '#040406');
-        document.body.style.setProperty('--history-lines', '#1b1b22');
-        document.body.style.setProperty('--box-color', '#060606');
-        document.body.style.setProperty('--boxmain-bordercolor', '#d487d4');
-        document.body.style.setProperty('--button-color', '#040406');
-        document.body.style.setProperty('--hover-color', '#1b1b22');
-        document.body.style.setProperty('--buttonbuy-color', '#040406');
-        document.body.style.setProperty('--buildings-canbuy-color', '#414162');
-        document.body.style.setProperty('--buildings-hover-color', '#4f4f76');
-        document.body.style.setProperty('--blessings-canbuy-color', '#34344d');
-        document.body.style.setProperty('--blessings-hover-color', '#48486c');
-        document.body.style.setProperty('--tab-color', 'black');
-        document.body.style.setProperty('--singtab-color', '#002');
-        document.body.style.setProperty('--hoversing-color', '#00007d');
-        DOMCacheGetOrSet('corruptionStatsLoadouts').style.backgroundColor = '#060606'
-        DOMCacheGetOrSet('corruptionStatsLoadouts').style.borderColor = '#dd8f00'
-        DOMCacheGetOrSet('actualPotionShop').style.backgroundColor = '#060606'
-        DOMCacheGetOrSet('actualPotionShop').style.borderColor = '#dd0'
-        DOMCacheGetOrSet('exportgame').style.backgroundColor = 'black' //Special cases
-        DOMCacheGetOrSet('importFileButton').style.backgroundColor = 'black'
-
-        themeButton.textContent = 'Darker Mode';
+export const theme: Theme = { //Will be really usefull, if custom themes will be added
```

Comment:
can you expand on the comment or remove it?

Reply:
You mean to explain how to use it?

Reply:
Into this object all colors can be moved, and then localy saved. Allowing to make easier custom themes